ros2doctor/README.md

```diff
@@ -0,0 +1,30 @@
+# ros2doctor
+
+This folder contains the source code of ros2doctor.
+It is part of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro.
+ros2doctor command line tool is similar to roswtf in ROS system.
```

Comment:
```suggestion
`ros2doctor` is similar to `roswtf` from ROS 1.
```
I'm not sure if this is true or not, but do you mean it's similar to a tool from ROS 1? If that's not right then ignore this!